### PR TITLE
Added PyDocs to WBValue, Image, Table, JoinedTable, Classes, BoundingBoxes2d, and ImageMask

### PR DESCRIPTION
**Pull request by tssweeney**
Added relevant PyDocs to WBValue, Image, Table, JoinedTable, Classes, BoundingBoxes2d, and ImageMask which relate to artifacts. In particular made sure their constructors and to/from JSON methods are properly doced.

exclusively a doc edit - no code introduced

### CODE REVIEW

File: wandb/data_types.py [hunk 1]
```diff
@@ -375,7 +418,7 @@ def seq_to_json(self, seq, run, key, step):
 
 
 class Table(Media):
-    """This is a table designed to display small sets of records.
+    """This is a table designed to display sets of records.
```
Comment (shawnlewis) [inline wandb/data_types.py:421]: I think we should say "store" instead of "display"

File: wandb/data_types.py [hunk 2]
```diff
@@ -1101,15 +1164,39 @@ class Classes(Media):
     artifact_type = "classes"
 
     def __init__(self, class_set):
+        """Classes is holds class metadata intended to be used in concert with other objects when visualizing artifacts
```
Comment (shawnlewis) [inline wandb/data_types.py:1167]: Type here: s/is//

File: wandb/data_types.py [hunk 3]
```diff
@@ -1101,15 +1164,39 @@ class Classes(Media):
...
+        Args:
+            class_set (list): list of dicts in the form of {"id":int|str, "name":str}
+        """
         super(Classes, self).__init__()
         self._class_set = class_set
         # TODO: validate
 
     @classmethod
     def from_json(cls, json_obj, source_artifact):
+        """Deserialize a `json_obj` into it's class representation. If additional resources were stored in the
```
Comment (shawnlewis) [inline wandb/data_types.py:1178]: Do we need to repeat this same comment everywhere?

**Review by shawnlewis — APPROVED** [review]

File: wandb/data_types.py (hunk 2 above)
Comment (tssweeney) [reply wandb/data_types.py:1154]: incoming

File: wandb/data_types.py (hunk 1 above)
Comment (tssweeney) [reply wandb/data_types.py:427]: incoming

File: wandb/data_types.py (hunk 3 above)
Comment (tssweeney) [reply wandb/data_types.py]: I thought, yes, but further testing shows no. incoming change

**Pull request merged**